packages/dependency-profiles/src/index.js

```diff
@@ -0,0 +1,302 @@
+// This file was generated by 'update-profile.js'
+/* eslint-disable */
+module.exports = {
+  "0.68": {
+    "svg": {
+      "name": "react-native-svg",
+      "version": "12.3.0"
+    },
```

Comment:
Unsure if we need to specify this dependency as part of our profile. I added it in because we are pinned to a specific version of react-native-svg in our repo but that won't be the case forever. I'm considering removing it for now.

Reply:
I think eventually we might want/need it since a lot of our icons are SVGs. But yeah makes sense to not add it just yet!

Reply:
it's actually a part of the base dependency profile for 0.68 already, even https://github.com/microsoft/rnx-kit/blob/49412e1cc1677275cd3ecfafd960314a685b0693/packages/dep-check/src/presets/microsoft/profile-0.68.ts#L105 which to me is a bit odd but mostly because the mainline rn-svg is up to version 13.4.0

Reply:
I think these are mostly manually updated by people who have used it and know what works (The "magic" part of rep-check is just manual verification..). I guess nobody we know has tried to use rn-svg 13.4.0 yet so it hasn't been updated? FURN breaks with 12.40+ so we personally haven't upgraded. That probably informs the dependencies of our downstream partners.

Reply:
> I guess nobody we know has tried to use rn-svg 13.4.0 yet so it hasn't been updated?

This is mostly the case. Just from looking at the changelog, it also looks like they broke backwards compatibility in 13.x. 13.4 is supposed to restore support for react-native older than 0.68. If FURN does upgrade to 13.x, let us know and we can update the recommendations.